src/Sulu/Bundle/WebsiteBundle/Resources/js/containers/CacheClearToolbarAction/CacheClearToolbarAction.js

```diff
@@ -8,9 +8,14 @@ import {translate} from 'sulu-admin-bundle/utils';
 export default class CacheClearToolbarAction {
     static clearCacheEndpoint: string;
 
+    searchParams: Object;
```

Comment:
we called this `requestParameters` in nearly all other places. would like to keep this consistent